src/test/node/user.js

```diff
@@ -1,9 +1,2 @@
 import test from 'tape';
 
-test('Insert Mock Users test', (t) => {
```

Comment:
Is this removed from this PR or was this just carried over from another PR that's already merged?